### PR TITLE
RUBY-1695 Regenerate test certificates with SHA256 signatures

### PR DESCRIPTION
**Pull request by saghm**

### CODE REVIEW

File: spec/lite_spec_helper.rb [hunk 1]
```diff
@@ -24,7 +24,6 @@
 else
   SSL_CERTS_DIR = "#{CURRENT_PATH}/support/certificates"
   CLIENT_PEM = "#{SSL_CERTS_DIR}/client.pem"
-  CLIENT_PASSWORD_PEM = "#{SSL_CERTS_DIR}/password_protected.pem"
```
Comment (saghm) [inline spec/lite_spec_helper.rb]: This doesn't seem to be used anywhere, so I've deleted it

File: spec/support/certificates/ca.pem [hunk 2]
```diff
@@ -1,17 +1,22 @@
...
-GzAZBgkqhkiG9w0BCQEWDHJvb3RAbGF6YXJ1czCBnzANBgkqhkiG9w0BAQEFAAOB
-jQAwgYkCgYEA1xymeY+U/evUuQvxpun9moe4GopN80c1ptmaAHM/1Onwaq54Wt27
-nl1wUVme3dh4DdWviYY7mJ333HVEnp/QhVcT4kQhICZqdgPKPdCseQW3H+8x6Gwz
-hrNRBdz0NkSoFxDlIymfy2Q2xoQpbCGAg+EnRYUTKlHMXNpUDLFhGjcCAwEAATAN
-BgkqhkiG9w0BAQUFAAOBgQDRQB3c/9osTexEzMPHyMGTzG5nGwy8Wv77GgW3BETM
-hECoGqueXLa5ZgvealJrnMHNKdj6vrCGgBDzE0K0VdXc4dLtLmx3DRntDOAWKJdB
-2XPMvdC7Ec//Fwep/9emz0gDiJrTiEpL4p74+h+sp4Xy8cBokQ3Ss5S9NmnPXT7E
-qQ==
------END CERTIFICATE-----
+-----BEGIN CERTIFICATE-----
```
Comment (saghm) [inline spec/support/certificates/ca.pem]: The certificates were generated from scratch with x509gen

File: spec/support/certificates/crl.pem [hunk 3]
```diff
@@ -1,10 +1,12 @@
 -----BEGIN X509 CRL-----
-MIIBazCB1QIBATANBgkqhkiG9w0BAQUFADCBkjELMAkGA1UEBhMCVVMxETAPBgNV
-BAgMCE5ldyBZb3JrMRYwFAYDVQQHDA1OZXcgWW9yayBDaXR5MQ4wDAYDVQQKDAUx
-MEdlbjEPMA0GA1UECwwGS2VybmVsMRowGAYDVQQDDBFNeSBDZXJ0IEF1dGhvcml0
-eTEbMBkGCSqGSIb3DQEJARYMcm9vdEBsYXphcnVzFw0xMjEyMTIxODQ3NDFaFw00
-MDA0MjgxODQ3NDFaoA4wDDAKBgNVHRQEAwIBCzANBgkqhkiG9w0BAQUFAAOBgQAu
-PlPDGei2q6kdkoHe8vmDuts7Hm/o9LFbBmn0XUcfHisCJCPsJTyGCsgnfIiBcXJY
-1LMKsQFnYGv28rE2ZPpFg2qNxL+6qUEzCvqaHLX9q1V0F+f8hHDxucNYu52oo/h0
-uNZxB1KPFI2PReG5d3oUYqJ2+EctKkrGtxSPzbN0gg==
+MIIBzzCBuAIBATANBgkqhkiG9w0BAQsFADB1MRcwFQYDVQQDEw5SdWJ5IERyaXZl
```
Comment (saghm) [inline spec/support/certificates/crl.pem]: The CRL's were manually generated since x509gen apparently no longer will do so

**Comment by p-mongo** [comment]
Is this ready for merge?

**Comment by saghm** [comment]
Sorry, forgot to add you as a reviewer on this

**Comment by p-mongo** [comment]
Can you link to documentation/document how these certificates were generated please?

**Comment by saghm** [comment]
As mentioned in the comments (and as is standard practice with the drivers team), these were generated with x509gen. I'll slack you the link (it's not a publicly accessible site, so there's no use leaving it here)

**Pull request merged**